### ISSUE TITLE
NVDA not announcing aria-live region - polite with aria-relevant=additions - at all

### ISSUE THREAD

**Issue by ros3cin** [nvaccess/nvda#15857]
### Steps to reproduce:

1. Go to https://jsfiddle.net/j4e6ugdw/1/
2. Activate NVDA
3. Click on the button to add announcements

### Actual behavior:

Nothing is announced

### Expected behavior:

NVDA should announce the recently added elements

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2023.3.0.29780

#### Windows version:
Windows 11 Enterprise 23H2 Build 22631.2715

#### Name and version of other software in use when reproducing the issue:
Edge Version 119.0.2151.93 (Official build) (64-bit) ❌
Chrome Version 119.0.6045.160 (Official Build) (64-bit) ❌
Firefox Version 119.0.1 (64-bit) ❌ 

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, no add-ons.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by sandersaspera** [comment]
Same for us. we're using custom components that work on other screenreaders but need a "hack" for NVDA. See also https://github.com/ionic-team/ionic-framework/blob/a3cd204f616606ccffc35082655e55fdfb19fe28/core/src/components/toast/toast.tsx#L641 which mentions this problem and how to circumvent it

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by seanbudd** [comment]
No updates

**Comment by Adriani90** [comment]
Jaws is also doing weird things here, it announces 1 minutes and 50 seconds multiple times in a row.
Narator seem to work best with this minimal example.
Since this also affects Firefox, @jcsteh any idea why NVDA does not consider aria-relevant=additions atribute? Is this intentional?
